### PR TITLE
[FLINK-14984] Remove old WebUI

### PR DESCRIPTION
**Pull request by lamber-ken**
## What is the purpose of the change

Flink 1.9 shipped with a new UI, with the old one being kept around as a backup in case something wasn't working as expected. So remove old WebUI.

## Brief change log

  - remove `old-version` folder in flink-runtime-web module
  - remove link at the new webui nav bar 

## Verifying this change

This change is code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)

### CODE REVIEW

**Comment by zentol** [comment]
The licensing information in the source [NOTICE](https://github.com/apache/flink/blob/master/NOTICE) must be updated to reflect the removal.
I _think_ this includes all the MIT stuff, nvd3, d3 and partially font-awesome.

**Comment by lamber-ken** [comment]
> The licensing information in the source [NOTICE](https://github.com/apache/flink/blob/master/NOTICE) must be updated to reflect the removal.
> I _think_ this includes all the MIT stuff, nvd3, d3 and partially font-awesome.

right, thanks for reminding.

File: flink-runtime-web/src/main/resources/META-INF/NOTICE [hunk 1]
```diff
@@ -4,47 +4,6 @@ Copyright 2014-2019 The Apache Software Foundation
...
-- d3:3.5.12
-
-This project bundles the following dependencies under SIL OFL 1.1 license. (https://opensource.org/licenses/OFL-1.1)
-See bundled license files for details.
-
-- font-awesome:4.5.0 (Font)
-
---------------------------------------------------------------------------------------------------------------------
-
 The new frontend bundles the following dependencies:
```
Comment (TisonKun) [inline flink-runtime-web/src/main/resources/META-INF/NOTICE]: shall we still use "new" here?

Comment (lamber-ken) [reply flink-runtime-web/src/main/resources/META-INF/NOTICE]: good catch

**Comment by zentol** [comment]
[This line](https://github.com/apache/flink/blob/6256b11edb5fc53bc655584591c6b3cd7349fdda/NOTICE#L43) also must be removed.

**Comment by zentol** [comment]
As well as a bunch of `LICENSE.xxx` files in `licenses` and `flink-runtime-web/src/main/resources/META-INF/licenses`. Double-check with the `NOTICE` file entries for the new WebUI to figure out which need to be removed.

**Comment by lamber-ken** [comment]
> As well as a bunch of `LICENSE.xxx` files in `licenses` and `flink-runtime-web/src/main/resources/META-INF/licenses`. Double-check with the `NOTICE` file entries for the new WebUI to figure out which need to be removed.

ok, thanks

**Comment by lamber-ken** [comment]
@flinkbot run travis

**Comment by vthinkxie** [comment]
Hi @lamber-ken the `InterationTwoTone` in the app.module.ts could also be removed after the old UI removed

**Comment by lamber-ken** [comment]
> InterationTwoTone

done.

**Comment by lamber-ken** [comment]
> I think you can actually remove all license files in flink-runtime-web. The licenses for all dependencies are directly included in the NOTICE file; I can't understand why some have a dedicated license file.

Hi, compare to `flink-runtime`, `flink-mesos` module, license files also exist.

https://github.com/apache/flink/tree/master/flink-mesos/src/main/resources/META-INF
https://github.com/apache/flink/tree/master/flink-runtime/src/main/resources/META-INF

**Comment by zentol** [comment]
We can remove all the license files in this case.

Usually, (literally in all modules _except_ flink-runtime-web) the NOTICE files lists dependencies, and the licenses are bundled separately in the `licenses` directory.
For the WebUI we used some generation tool for collecting the licenses and put that directly into the NOTICE.
That's why the licenses directory isn't necessary in this case.

**Comment by lamber-ken** [comment]
> We can remove all the license files in this case.
> 
> Usually, (literally in all modules _except_ flink-runtime-web) the NOTICE files lists dependencies, and the licenses are bundled separately in the `licenses` directory.
> For the WebUI we used some generation tool for collecting the licenses and put that directly into the NOTICE.
> That's why the licenses directory isn't necessary in this case.

Thanks for you help, I had remove all license files.

**Pull request merged**